openassessment/xblock/test/test_config_mixin.py

```diff
@@ -105,31 +104,6 @@ def test_all_files_urls_enabled(
             'is_fetch_all_urls_waffle_enabled',
         )
 
-    @ddt.data(
-        *list(itertools.product([True, False], repeat=3))
-    )
-    @ddt.unpack
-    @mock.patch('openassessment.xblock.config_mixin.import_waffle_switch', autospec=True)
-    @mock.patch('openassessment.xblock.config_mixin.import_course_waffle_flag', autospec=True)
-    def test_mobile_support_enabled(
-            self, waffle_switch_input, waffle_flag_input, settings_input, mock_waffle_flag, mock_waffle_switch
-    ):
```

Comment:
Instead of removing this test completely, could you instead test the feature flag [as is done here](https://github.com/edx/edx-ora2/blob/c12b4851cfbbaea43aee91ddf59d7fcba3dea8b3/openassessment/xblock/test/test_config_mixin.py#L129-L131)?

Want to verify both that this feature is off by default, and on when the feature flag is present is the settings.

Reply:
Thanks for re-adding that test! It can be a bit simpler though, since it's not as complex as the other tests:
```python
    @ddt.data(
        (True, True),
        (False, False),
        (None, False),
    )
    @ddt.unpack
    def test_mobile_support_enabled(
            self, settings_input, expected_output
    ):
        """
        The mobile support is expected to be enabled only if:
          1) The settings.FEATURES['ENABLE_ORA_MOBILE_SUPPORT'] value is True.
        """
        my_block = MockBlock()
        settings_feature_key = FEATURE_TOGGLES_BY_FLAG_NAME[MOBILE_SUPPORT]
        with self.settings(FEATURES={settings_feature_key: settings_input}):
            self.assertEqual(expected_output, my_block.is_mobile_support_enabled)
```